zap/src/main/java/org/parosproxy/paros/core/scanner/Alert.java

```diff
@@ -223,6 +225,7 @@ public static Source getSource(int id) {
     private URI msgUri = null;
     private Source source = Source.UNKNOWN;
     private String alertRef = "";
+    private Map<String, String> tags = new HashMap<>();;
```

Comment:
- It would be better to use `Collections.emptyMap()` to avoid garbage, or leave as null.
 - The second semicolon could be removed.